WordPress/Classes/ViewRelated/Blog/EditSiteViewController.m

```diff
@@ -445,7 +445,7 @@ - (void)togglePushNotifications:(id)sender
         NSMutableArray *mutedBlogsArray = [[mutedBlogsDictionary objectForKey:@"value"] mutableCopy];
         NSMutableDictionary *updatedPreference;
 
-        NSNumber *blogID = [self.blog isWPcom] ? self.blog.blogID : [self.blog jetpackBlogID];
+        NSNumber *blogID = [self.blog isWPcom] ? self.blog.blogID : self.blog.jetpack.siteID;
```

Comment:
This logic should be handled by the class `Blog` itself.  As an added bonus the next set of changes (below) in this diff wouldn't need to repeat this code.